docs/field-descriptions.yaml

```diff
@@ -67,6 +67,9 @@ data_path: Relative path to file or directory with instrument data. Downstream p
   will depend on filename extension conventions.
 data_precision_bytes: Numerical data precision in bytes
 description: Free-text description of this assay.
+desi_solvent: TODO
+desi_solvent_flow_rate: TODO
+desi_solvent_flow_rate_unit: TODO
```

Comment:
```suggestion
desi_solvent_flow_rate_unit: Units of the solvent flow rate.
```

Reply:
@cebriggs7135 -- As discussed, changes should not be suggested in `docs`, only in `src`.